packages/shell-api/src/database.ts

```diff
@@ -26,13 +26,14 @@ import {
   getBadge,
 } from './helpers';
 
-import type {
-  ChangeStreamOptions,
-  CommandOperationOptions,
-  CreateCollectionOptions,
-  Document,
-  WriteConcern,
-  ListCollectionsOptions,
+import {
+  type ChangeStreamOptions,
+  type CommandOperationOptions,
+  type CreateCollectionOptions,
+  type Document,
+  type WriteConcern,
+  type ListCollectionsOptions,
+  ReadPreference,
```

Comment:
Where is `ReadPreference` being used? Generally speaking, we don’t want to import runtime values from the Node.js driver via `@mongosh/service-provider-core`, only types

Reply:
Ah yes, you are totally right, this is a leftover. Fixing it!